src/main/java/org/kairosdb/datastore/cassandra/CQLBatch.java

```diff
@@ -190,4 +200,12 @@ public void submitBatch()
 			m_batchStats.addDatapointsBatch(dataPointBatch.size());
 		}
 	}
+
+	public Collection<String> getMetricNames() {
+		return m_metricNames;
```

Comment:
You should probably wrap these with `Collections.unmodifiableCollection()` to prevent the caller from mutating the internals of this class.